routers/quagga.php

```diff
@@ -62,6 +62,11 @@ protected function build_as($parameter) {
     $parameter = '^'.$parameter.'_';
     return $this->build_aspath_regexp($parameter);
   }
+
```

Comment:
Not required if the IPv6 specific AS path regexp function is removed and its logic included in the existing AS path function.